tox.ini

```diff
@@ -75,8 +75,7 @@ commands =
     python run_pylint.py
 deps =
     pep8
-    -ehg+https://bitbucket.org/logilab/pylint@33e334be064c#egg=pylint
-    logilab-common<=0.63.0  # Hack for pylint upstream
+    pylint
     unittest2
```

Comment:
In that case you'll have an extra surprise when we'll move to GitHub. ;-) It's planned after the release, so should be fairly soon (not 6 months "soon"!)